src/test/java/com/wikia/webdriver/common/core/MailFunctions.java

```diff
@@ -103,9 +103,9 @@ public static void deleteAllEmails(String userName, String password) {
       }
       store.close();
     } catch (NoSuchProviderException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)